workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,127 @@
+
+# Create New Widget
+
+Navigate to the dashboard you would like to add a new widget to and select __Add Widget__ and __Create Widget__:
+
+![](https://lh7-us.googleusercontent.com/Qn2pnY24-2CuG7moVbGCi4UqO0aoIMdqNjsYaSipLHo3gfUOhD_z0vchbVyMNrN3zb6gPZeCAkkoJWmndzgqtcZVVFz9jzJYD_gkhP3qWNytupLefOGdMGLhd5Z9SjTvxqXbMmM1x6XYF0QzXK9LNn8)
+
+![](https://lh7-us.googleusercontent.com/Oe2a3IYXnQ4JTYXRbzMe9c7RZSHXFySFLJGFZCxWhJIFIvlNM2hOmGgQdCapl2h8Hk5-FhEWRwALQxZ1lJ2BGJtnvQnXA_r0TCqX5o3OeXwJnAqQrR1wkm77-uZLmL5V7QTiGpPr4Uhc6SRvjaO3Yjg)
+
+### Step 1: Select Chart Type
+
+Choose from one of the following types of charts for your widget:
+
+![](https://lh7-us.googleusercontent.com/LyMx_1CWIjqGEdzQYF_0XveQ3v-NvYJHLFA3N64iQfmfIx7KisKS0v2zx2VdMlB2caX9bAhYJUA9VULXsE8zGZHNH6XiBYbi0tjhlYTx8Uv5uBVKi80uDlceM9TUIGlZATlxwD6Pa1mJywRFU3XOYUQ)
+
+### Step 2: Create Query
+
+Create your query logic in the Metrics Section. Select the specific metric you would like to query, aggregate, and filter. Specify the interval at which data is ingested, create metric aliases to easily interpret what is displayed within the widget, and filter out unwanted data within the query.
+
+#### Group By
+
+The Group By field is the outer most aggregation made in your query. The way in which you aggregate a metric is contingent on the initial metric you have selected for your widget. Group your initial metric with resource attributes, metric attributes, or another metric based on the selected metric. The below example is grouping the `container.name` metric by all unique instances of the `container.name` metric:
+
+![](https://lh7-us.googleusercontent.com/lJ_3zm4C37Zgv8oWBRza4Z9-BbdJdlXD-lxbFq2qfGD579JQ9tNR_1MbC0zLLLW9ufQ-MQQa1_nwD4tg5PG-9QTpCDdsX5rE4UDIN-y9sILxyPlCHF8C8FcYv1od9VqK2ZACHS1NBAQFDk6yuOLgRzM)
+
+The following table displays all available methods for aggregating your chosen metrics:
+
+| __Aggregation Type__ | __Description__                                                |
+| :------------------- | :------------------------------------------------------------ |
+| __Average By__       | Average of all values in selected time range for given metric |
+| __Sum By__           | Sum of all values in selected time range for given metric |
+| __Minimum By__       | Minimum of all values in selected time range for given metric |
+| __Maximum By__       | Maximum of all values in selected time range for given metric |
+| __Any By__           | Gathers all possible values in selected time range for given metric __and/or__ attribute |
+| __Unique By__        | Count of unique values in selected time range for given metric __and/or__ attribute |
+| __Count By__         | Count of all values in selected time range for given metric __and/or__ attribute |
+| __Group By__         | Groups all values in selected timerange for given metric __and/or__ attribute |
+
+#### Inner Value Aggregation
```

Comment:
Let's call this "Aggregated By" since that's what it's called in the app. Or, let's change what the app phrasing is if we think there's a more clear term

Reply:
One other note on this, let's use an example where the inner value is a sum and the outer value is an average as that's usually the most confusing. Ed would probably be best to help crystalize an example here. There's probably some language and potentially diagrams from his doc we can bring in here. 

https://docs.google.com/document/d/1MnibcNnVGjdRzZkdTbE5AmBI-o7Q7wgK7KcTZnkMo0o/edit#heading=h.uixg7dnhcob8